src/patternfly/components/Expandable/expandable.scss

```diff
@@ -18,20 +20,22 @@
   // Expandable expanded
   --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--md);
 
-  &.pf-m-expanded {
-    --pf-c-expandable__toggle--Color: var(--pf-c-expandable__toggle--m-expanded--Color);
-
-    .pf-c-expandable__toggle-icon {
-      transform: rotate(90deg);
-    }
-  }
-
   .pf-c-expandable__toggle {
     padding: var(--pf-c-expandable__toggle--PaddingTop) var(--pf-c-expandable__toggle--PaddingRight) var(--pf-c-expandable__toggle--PaddingBottom) var(--pf-c-expandable__toggle--PaddingLeft);
     font-weight: var(--pf-c-expandable__toggle--FontWeight);
     color: var(--pf-c-expandable__toggle--Color);
     border: none;
 
+    &:focus,
+    &.pf-m-expanded {
```

Comment:
the expanded class is applied to the component, not the `.pf-c-expandable__toggle` element